### PR TITLE
Add suggestion to match top-level ID with filename

### PR DESCRIPTION
**Pull request by brunchboy**
Based on a recommendation from @KOLANICH

Also fixed a typo I noticed while searching to see if this was
already explained anywhere.

### CODE REVIEW

File: ksy_style_guide.adoc [hunk 1]
```diff
@@ -106,7 +106,7 @@ different purposes.
 
 For top-level meta section, use the following order of keys:
 
-. `id` — main identifier
+. `id` — main identifier, should match the name of the `.ksy` file
```
Comment (GreyCat) [inline ksy_style_guide.adoc]: Totally makes sense, but please use `MUST` instead of `should`, as this is actually a pretty hard requirement.

Comment (GreyCat) [reply ksy_style_guide.adoc]: (and it is `MUST`, not `must`, as we're reffering to RFC 2119 terms).

Comment (brunchboy) [reply ksy_style_guide.adoc]: Certainly! This change has been added to the pull request.

**Comment by GreyCat** [comment]
Thanks!

**Pull request merged**